tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -11,133 +11,211 @@
-@pytest.mark.dimensions("us-west-2", "c5.xlarge", "centos8", "*")
-@pytest.mark.dimensions("us-east-1", "c5.xlarge", "ubuntu1804", "*")
-@pytest.mark.dimensions("us-gov-west-1", "c5.xlarge", "ubuntu1804", "*")
-@pytest.mark.dimensions("cn-northwest-1", "c4.xlarge", "alinux2", "*")
-def test_createami(region, os, instance, request, pcluster_config_reader, vpc_stack, architecture):
-    """Test createami for given region and os"""
-    cluster_config = pcluster_config_reader()
 
-    # Get base AMI
-    # remarkable AMIs are not available for ARM yet
-    base_ami = retrieve_latest_ami(region, os, ami_type="remarkable", architecture=architecture)
-
-    # Networking
-    vpc_id = vpc_stack.cfn_outputs["VpcId"]
-    networking_args = ["--vpc-id", vpc_id, "--subnet-id", vpc_stack.cfn_outputs["PublicSubnetId"]]
-
-    # Custom Cookbook
-    custom_cookbook = request.config.getoption("createami_custom_chef_cookbook")
-    custom_cookbook_args = [] if not custom_cookbook else ["-cc", custom_cookbook]
-
-    # Custom Node
-    # inject PARALLELCLUSTER_NODE_URL into packer environment
-    custom_node = request.config.getoption("createami_custom_node_package")
-    env = None
-    if custom_node:
-        env = environ.copy()
-        env["PARALLELCLUSTER_NODE_URL"] = custom_node
-
-    # Instance type
-    pcluster_version_result = run_command(["pcluster", "version"])
-    instance_args = (
-        [] if version.parse(pcluster_version_result.stdout.strip()) < version.parse("2.4.1") else ["-i", instance]
-    )
+def _assert_build_tag(image_id):
+    logging.info("Check the build tag is present as specified in config file.")
+    stack_list = boto3.client("cloudformation").describe_stacks(StackName=image_id).get("Stacks")
+    logging.info(stack_list)
+    assert_that(len(stack_list)).is_equal_to(1)
+    stack_tags = stack_list[0].get("Tags")
+    logging.info(stack_tags)
+    assert_that(stack_tags).contains({"Key": "dummyBuildTag", "Value": "dummyBuildTag"})
 
-    pcluster_createami_result = run_command(
-        ["pcluster", "createami", "-ai", base_ami, "-os", os, "-r", region, "-c", cluster_config.as_posix()]
-        + custom_cookbook_args
-        + instance_args
-        + networking_args,
-        env=env,
-        timeout=7200,
-    )
-
-    stdout_lower = pcluster_createami_result.stdout.lower()
-    assert_that(stdout_lower).contains("downloading https://{0}-aws-parallelcluster.s3".format(region))
-    assert_that(stdout_lower).does_not_contain("chef.io/chef/install.sh")
-    assert_that(stdout_lower).does_not_contain("packages.chef.io")
-    assert_that(stdout_lower).contains("thank you for installing cinc client")
-    assert_that(stdout_lower).contains("starting cinc client")
-    assert_that(stdout_lower).does_not_contain("no custom ami created")
 
+def _assert_image_tag_and_volume(image_id):
+    logging.info("Check the image tag is present as specified in config file.")
+    image_list = (
```

Comment:
I think here we have the alternative to use describe-images command to see if it's correctly tagged. In this way we're also testing the describe image command.